index.html

```diff
@@ -40,6 +38,30 @@ <h3>I’am a Software Developer</h3>
                 <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
             </ul>
         </section>
+        <section class="graphics">
+                <div class="graph-iten-one">
+                    <img src="./images/corner-img.svg" alt="" >
+                </div>
+                <div class="top-black">
+                    <img src="./images/top-black.svg" alt="">
+                </div>
+                <div class="top-white">
+                    <img src="./images/top-white.svg" alt=""> <span><img src="./images/top-small.svg" alt="" id="overlay-black"></span>
+                </div>
+                <img src="./images/curve-img.svg" alt="" id="curve-img">
+                <div class="graph-big-img-two">
+                    <img src="./images/big-img-two.svg" alt="" class="two-svg">
+                </div>
+               
+                <div class="graph-big-img">
+                    <img src="./images/big-img.svg" alt="" ><span><img src="./images/second-small.svg" alt="" id="overlay-big"></span>
+                </div>
+                <img src="./images/down-one.svg" alt="" class="down-img">
+                <img src="./images/down-two.svg" alt="" class="down-img-two">
```

Comment:
This section is not in the design. Please follow the design and add background images as seen in the given design.